src/utils/u60_representation.nr

```diff
@@ -57,6 +58,29 @@ impl<let N: u32, let NumSegments: u32> std::convert::From<[Field; N]> for U60Rep
     }
 }
 
+// impl<let N: u32, let NumSegments: u32> std::convert::From<Field> for U60Repr<N, NumSegments> {
```

Comment:
this would be my preferred way of doing the casting for `U60Repr` but the issue is as soon as I have the conversion like this, I can not use `U60repr::from()` anymore. because it does not know the type generic. @TomAFrench any suggestions on how to fix this?